### PR TITLE
[DEV-5487] Add new fields to IDV account endpoint

### PR DESCRIPTION
**Pull request by kbard**
**Description:**
Add DEF code, outlay, and period columns to `/api/v2/idvs/funding/` endpoint.

Turns out this was an oversight in the ticket.  Thankfully, Dr @jonhill13 discovered it before it was too late.

Paired with:  https://github.com/fedspendingtransparency/usaspending-website/pull/1773

**Requirements for PR merge:**

1. [x] Tests unaffected somehow
2. [x] API documentation updated
3. [x] Necessary PR reviewers:
    - [ ] Backend
    - [ ] Frontend @jonhill13 
4. [x] Materialized views unaffected
5. [x] Frontend impact assessment completed
6. [x] Data validation completed
7. [x] No Operations ticket required
8. [x] Jira Ticket [DEV-5487](https://federal-spending-transparency.atlassian.net/browse/DEV-5487):
    - [x] Link to this Pull-Request

### CODE REVIEW

File: usaspending_api/api_contracts/contracts/v2/idvs/funding.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ List File C financial data for an IDV award's descendant contracts
                     Program activity name, program activity code
                 + `reporting_agency_name`
                 + `reporting_fiscal_date`
-                    Reporting fiscal year, reporting fiscal period
+                    Reporting fiscal year, reporting fiscal month
```
Comment (jonhill13) [inline usaspending_api/api_contracts/contracts/v2/idvs/funding.md:36]: are we able to sort by `gross_outlay_amount` and `disaster_emergency_fund_code `?

Comment (kbard) [reply usaspending_api/api_contracts/contracts/v2/idvs/funding.md:38]: What are you talking about?  They're right there!  😛

Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/idvs/funding.md:38]: @kbard where? I don't see them in the sort enum list either

Comment (kbard) [reply usaspending_api/api_contracts/contracts/v2/idvs/funding.md:38]: image

Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/idvs/funding.md:38]: image

Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/idvs/funding.md:38]: We can chalk that up to github's improved U/I hiding commits

**Pull request merged**